docs/src/pages/demos/selection-controls/Switches.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Switch from '@material-ui/core/Switch';
+
+function Switches() {
+  const [state, setState] = React.useState({
+    checkedA: true,
+    checkedB: true,
+  });
+
+  const handleChange = (name: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Shouldn't this be memoized with `useMemo` to prevent Switch from re-rendering?

Reply:
Our components are not memoized by default. Can't use `useMemo` here anyway it's called dynamically.